packages/hardhat-core/src/internal/util/download.ts

```diff
@@ -1,5 +1,7 @@
 import fs from "fs";
 import fsExtra from "fs-extra";
+import { HttpsProxyAgent } from "https-proxy-agent";
+import type { RequestInit as FetchOptions } from "node-fetch";
```

Comment:
Manually defining `FetchOptions` wasn't necessary, it is better to import it from the module.

Reply:
Ah. I think I was a bit confused on that because of the `const { default: fetch } = await import("node-fetch");` which seemed strange as to where to put the import. Thank you!